### PR TITLE
[ADD] hr_employee_benefit_exemption

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
https://github.com/OCA/hr/pull/82 must pass first

**Comment by ghost** [comment]
@bwrsandman

**Comment by ghost** [comment]
@mdietrichc2c @charbeljc
I would also appreciate your reviews for this modules, as well as https://github.com/OCA/hr/pull/106 and https://github.com/OCA/hr/pull/105.

File: hr_employee_benefit_exemption/models/hr_salary_rule.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+
+class HrSalaryRule(orm.Model):
+    _inherit = 'hr.salary.rule'
+
+    def _filter_benefits(self, cr, uid, ids, payslip, context=None, **kwargs):
+        """ Remove all benefits that are exempted from a deduction.
+        """
+        benefits = super(HrSalaryRule, self)._filter_benefits(
+            cr, uid, ids, payslip, **kwargs)
```
Comment (mdietrichc2c) [inline hr_employee_benefit_exemption/models/hr_salary_rule.py]: `context` is not passed to super()? Is it wanted?

Comment (ghost) [inline hr_employee_benefit_exemption/models/hr_salary_rule.py]: nope, its an error ;-)

**Comment by ghost** [comment]
@mdietrichc2c Fixed the context issue.

**Comment by mdietrichc2c** [comment]
Okay, then LGTM :+1:

**Comment by charbeljc** [comment]
:+1:

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**